### PR TITLE
fix: add currency balance check to signed extras

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

File: runtime/testing-hydradx/src/lib.rs [hunk 1]
```diff
@@ -221,20 +222,6 @@ impl Contains<Call> for BaseFilter {
 			return false;
 		}
 
-		// filter transfers of LRNA to the omnipool account
-		if let Call::Tokens(orml_tokens::Call::transfer { dest, currency_id, .. })
```
Comment (jak-pan) [inline runtime/testing-hydradx/src/lib.rs:225]: Is this intentional as part of this PR?

File: runtime/testing-hydradx/src/lib.rs [hunk 2]
```diff
@@ -221,20 +222,6 @@ impl Contains<Call> for BaseFilter {
 			return false;
 		}
 
-		// filter transfers of LRNA to the omnipool account
```
Comment (lumir-mrkva) [inline runtime/testing-hydradx/src/lib.rs:224]: why did you remove this? some local changes slipped in?

File: runtime/testing-hydradx/src/lib.rs [hunk 3]
```diff
@@ -696,6 +683,19 @@ impl pallet_claims::Config for Runtime {
 
 impl pallet_genesis_history::Config for Runtime {}
 
+pub struct NoSpotPriceProvider;
```
Comment (lumir-mrkva) [inline runtime/testing-hydradx/src/lib.rs:686]: looks like you reverted some changes as well

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]
